admin/app/adapters/session.js

```diff
@@ -42,6 +42,10 @@ export default class SessionAdapter extends ApplicationAdapter {
       const url = this.urlForUpdateRecord(snapshot.id, type.modelName, snapshot) + '/certification-officer-assignment';
       return this.ajax(url, 'PATCH');
     }
+    if (snapshot.adapterOptions.unfinalize) {
```

Comment:
Suggestion: placer ce commit avant le précédent


Bonus: renommer le commit "admin: add unfinalize option in sessio adapter "